### PR TITLE
feat: Add helpers to UI and API for managing file uploads

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
Putting this up for initial feedback. I can probably get some tests working for the API field, though I'm not sure the same is easily said about the UI.

Right now I'm QA'ing it with some unpushed changes in getsentry for _admin.

**Comment by dcramer** [comment]
Example here (its just an ugly file field)

image

Payload is serialized as so:

image

**Comment by dcramer** [comment]
hello bot do not close me

**Comment by dcramer** [comment]
Add some basic unit tests for the field class on server-side

**Comment by dcramer** [comment]
https://github.com/getsentry/getsentry/tree/feat/policy-admin-edit

File: docs-ui/stories/components/form-fields.stories.js [hunk 1]
```diff
@@ -201,6 +202,14 @@ export const _RadioField = () => (
   </Form>
 );
 
+export const __FileField = () => (
+  <Form>
+    <FileField name="field" label="File Field" />
+  </Form>
+);
+
+__FileField.storyName = 'File';
```
Comment (evanpurkhiser) [inline docs-ui/stories/components/form-fields.stories.js:211]: Why two `__` cc @vuluongj20 ?

**Review by evanpurkhiser — APPROVED** [review]
looks good

File: docs-ui/stories/components/form-fields.stories.js (hunk 1 above)
Comment (dcramer) [reply docs-ui/stories/components/form-fields.stories.js:211]: idk the file makes no sense there are weird patterns all over

Comment (evanpurkhiser) [reply docs-ui/stories/components/form-fields.stories.js:211]: Oh yeah you're right lol

**Comment by dcramer** [comment]
plz someone find these changes if you ever refactor avatar uploader

**Pull request merged**